### PR TITLE
DSC-9978 - Complete transition to GHA

### PR DESCRIPTION
**Pull request by nickshoust-wf**
## Summary

Finish migration to GHA.
- Remove Dockerfile
- Remove skynet.yaml
- Generate an SBOM 

## Testing
 - CI passing
 - cyclone DX file attached as an artifact on the GHA build

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

**Comment by nickshoust-wf** [comment]
QA +1
https://github.com/Workiva/test_html_builder/actions/runs/8555898577?pr=46
Screenshot 2024-04-04 at 10 01 55 AM

File: .github/workflows/dart_ci.yaml [hunk 1]
```diff
@@ -23,9 +25,9 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        sdk: [ 2.18.7, stable, beta ]
+        sdk: [ stable, beta ]
```
Comment (robbecker-wf) [inline .github/workflows/dart_ci.yaml:28]: Can we add 2.19.6 while we're in here?

Comment (nickshoust-wf) [reply .github/workflows/dart_ci.yaml]: https://github.com/Workiva/test_html_builder/pull/46/commits/5cad8d31bbcc587ea8fc64e7cef719b80dd89ba0

**Review by robbecker-wf — APPROVED** [review]
+10 cyclone file attached, required GHA set

**Comment by robbecker-wf** [comment]
@Workiva/release-management-p

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**